src/main/java/com/minecolonies/colony/CitizenData.java

```diff
@@ -429,6 +429,7 @@ public AbstractJob getJob()
         }
         catch(ClassCastException exc)
         {
+            Log.logger.catching(exc);
```

Comment:
Because we are using the exception being thrown as a way of saying that job wasn't an instance of the type that you wanted. So we return null, we have handled the exception, but we don't need to log it.